### PR TITLE
Update UG, DG and UML diagram

### LINKED ISSUE #206: Update DG and UG
Update DG and UG

### PR DESCRIPTION
**Pull request by Solomon0519**
Closes #206 #217 #205

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -320,14 +320,10 @@ Examples:
...
-### Adding a property: `add`
-
-Adds a property to the property list.
 
-Format: `add n/LANDLORD_NAME p/PHONE_NUMBER a/ADDRESS s/ASKING_PRICE t/PROPERTY_TYPE`
+In property mode, you will be able to store and view the list of property's details.
 
-Examples:
-* `add n/John p/87152433 a/Paya Lebar s/200,000 t/Condominium`
+![buyer_mode display](images/PropertyModeDisplay.png)
```
Comment (NgZiXin) [inline docs/UserGuide.md:326]: property_mode_display

File: docs/UserGuide.md [hunk 2]
```diff
@@ -353,19 +358,36 @@ Examples:
 
 ### Locating Properties: `find`
 
-Find property based on its attributes.
+Find existing properties in the existing property list based on keywords.
```
Comment (NgZiXin) [inline docs/UserGuide.md:361]: based on either name or address keywords

File: docs/diagrams/StorageClassDiagram.puml [hunk 3]
```diff
@@ -42,23 +45,26 @@ StorageManager .up.|> Storage
...
 JsonBuyerListStorage .up.|> BuyerListStorage
+JsonMeetUpListStorage .up.|> MeetUpListStorage
 JsonPropertyListStorage .up.|> PropertyListStorage
 JsonBuyerListStorage ..> JsonSerializableBuyerList
+JsonMeetUpListStorage ..> JsonSerializableMeetUpList
 JsonPropertyListStorage ..> JsonSerializablePropertyList
 JsonSerializableBuyerList --> "*" JsonAdaptedBuyer
+JsonSerializableMeetUpList --> "*" JsonAdaptedMeetUp
 JsonSerializablePropertyList --> "*" JsonAdaptedProperty
 JsonAdaptedBuyer --> "*" JsonAdaptedTag
```
Comment (NgZiXin) [inline docs/diagrams/StorageClassDiagram.puml:65]: For property has JSONAdaptedAddeBuyer

File: docs/images/SwitchPropertyAfter.png
Comment (NgZiXin) [inline docs/images/SwitchPropertyAfter.png:1]: This one after switch mode, show some items in list

File: docs/images/PropertyModeDisplay.png
Comment (NgZiXin) [inline docs/images/PropertyModeDisplay.png:1]: Beside Address, bracket (No Duplicates)

File: docs/images/SwitchPropertyBefore.png
Comment (KennyLewi) [inline docs/images/SwitchPropertyBefore.png:1]: LGTM

**Review by KennyLewi — COMMENTED** [review]
LGTM

**Pull request merged**